modules/core/src/main/scala/higherkindness/droste/package.scala

```diff
@@ -9,13 +9,30 @@ import syntax.compose._
 
 object `package` {
 
+  /** Type Alias for `F[A] => A` */
   type Algebra[F[_], A]   = GAlgebra[F, A, A]
+
+  /** Type Alias `Coalgebra[F, A]` is a type alias for `GCoalgebra[F, A, A]`.
+    * This in turn reduces to a `A => F[A]` */
   type Coalgebra[F[_], A] = GCoalgebra[F, A, A]
 
+  /** Type Alias `AlgebraM[M, F, A] = GAlgebraM[M, F, A, A]`, i.e. a `GAlgebraM` where the
```

Comment:
Say this first
"A function `M[A] => F[A]`"
So that all scaladoc comments for these aliases are structured the same.